### PR TITLE
`New Post` GUI Challenges - Building a tooltip component

### LINKED ISSUE #8693: [A11y][CSS] Building a tooltip component [9/28/2022]
* Proposed title: Building a tooltip component
* Proposed by: argyle
* Related areas: Accessibility,CSS
* Content type: How-To/API Guide (https://documentation.divio.com/how-to-guides/)
* Deadline: 9/28/2022
* Deadline explanation: release with video
* [Draft](https://docs.google.com/document/d/1DWsJeS4zl3zJORGi8gn3FtS8L19Ad_CmavibA2uf4Ms/edit?pli=1#)
* Content summary: GUI Challenges on building a CSS only tooltip component

### PR DESCRIPTION
**Pull request by argyleink**
Fixes #8693 

To do:
- [ ] final reviews
- [ ] add final publishing date

### CODE REVIEW

File: src/site/content/en/blog/building-a-tooltip-component/index.md [hunk 1]
```diff
@@ -0,0 +1,717 @@
...
+layout: post
+title: Building a tooltip component
+subhead: A foundational overview of how to build a color-adaptive and accessible tooltip custom element.
+authors:
+  - adamargyle
+description: A foundational overview of how to build a color-adaptive and accessible tooltip custom element.
+date: 2022-10-24
+thumbnail: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/Mx5dDmELgbJottijzfwF.png
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/5XgqLJdeI33MDjvJu2te.png
+alt: A tilted grid of hotpink FABs.
```
Comment (rachelandrew) [inline src/site/content/en/blog/building-a-tooltip-component/index.md:11]: I don't think this alt relates to the image in use.

File: src/site/content/en/blog/building-a-tooltip-component/index.md [hunk 2]
```diff
@@ -0,0 +1,717 @@
...
+elements into web components if they don't want to. The browser will treat
+`<foo-bar>` just like a `<div>`. You could think of a custom element like a
+classname with less specificity. There's no JavaScript involved.
+
+```html
+<tool-tip>A tooltip</tool-tip>
+```
+
+This is like a div with some text inside. We can tie into the accessibility tree
+of capable screen readers by adding `[role="tooltip"]:
```
Comment (rachelandrew) [inline src/site/content/en/blog/building-a-tooltip-component/index.md:102]:
```suggestion
of capable screen readers by adding `[role="tooltip"]`.
```

**Review by rachelandrew — APPROVED** [review]
I just fixed some markup and made a suggestion re the hero alt.

File: src/site/content/en/blog/building-a-tooltip-component/index.md (hunk 1 above)
Comment (argyleink) [reply src/site/content/en/blog/building-a-tooltip-component/index.md:11]: nice catch, def a copy paste issue! i'll fix 🙂

**Pull request merged**